x/interchainstaking/keeper/callbacks.go

```diff
@@ -108,7 +108,7 @@ func RewardsCallback(k *Keeper, ctx sdk.Context, args []byte, query icqtypes.Que
 	// unmarshal request payload
 	rewardsQuery := distrtypes.QueryDelegationTotalRewardsRequest{}
 	if len(query.Request) == 0 {
-		return errors.New("attempted to unmarshal zero length byte slice (2)")
+		return errors.New("attempted to unmarshal zero length byte slice")
```

Comment:
Plz keep these numbers for easy-to-distinguish errors from each other
And hey @joe-bowman, why don't we have this line `"attempted to unmarshal zero length byte slice (5)"`, I see we numbered from 1 to 9 but there is a leak of number 5